rules/prefer-array-index-of.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+const {hasSideEffect, isParenthesized} = require('eslint-utils');
+const getDocumentationUrl = require('./utils/get-documentation-url');
+const methodSelector = require('./utils/method-selector');
+
+const MESSAGE_ID_FIND_INDEX = 'findIndex';
+const messages = {
+	[MESSAGE_ID_FIND_INDEX]: 'Use `.indexOf()`, rather than `.findIndex()`, when searching the index of an item.'
```

Comment:
```suggestion
	[MESSAGE_ID_FIND_INDEX]: 'Use `.indexOf()` instead of `.findIndex()` when looking for the index of an item.'
```

Reply:
I will do it as you want it. By the past, I used to use _instead of_, but I realized that it is ambiguous. When you say "x instead of y", you don't know what is good and what is bad. You can say: _"You suck at using `.findIndex()` instead of `.indexOf()`."_. _Rather than_ is not ambiguous. That's why I pretty like _prefer over_ too.

Reply:
It seems that `prefer-array-index-of.js.snap` needs to be updated but I don't know how.